docs/user_guides/fs/tags/tags.md

```diff
@@ -0,0 +1,138 @@
+# Tags 
+
+## Introduction
+
+Hopsworks feature store enables users to attach tags to artifacts, such as feature groups, feature views or training datasets.
+
+A tag is a `{key: value}` association which provides additional information about the data managed by Hopsworks. Tags allow organizations to customize the feature store metadata and build a governance layer on top of it. Hopsworks indexes tags and makes them searchable by users on the platform, increasing the discoverability of features and artifacts in the feature store.
+
+## Prerequisites
+
+Tags need need to respect a given schema. Before being able to attach a tag to an artifact you need to define a schema.
+Schemas can be define by Hopsworks administrator in the `Cluster settings` section of the platform. Schemas are defined globally across all projects. When users attach tags to an artifact, the tag will be validated against a specific schema. This allows tags to be consistent no matter the project or the team generating them. 
+
+!!! warning "Immutable"
+    Tag schemas are immutable. Once defined, a schema cannot be edited nor deleted. This is to prevent inconsistencies when attaching tags to different artifacts.
```

Comment:
tag schema

Reply:
Also, no need to justify our design choice here. This sentence is not necessary (IMO):
This is to prevent inconsistencies when attaching tags to different artifacts.